### DISCUSSION TITLE
Unit tests

### DISCUSSION THREAD

**Discussion by sjoness** [gionoa/ErgastAPI#43] [General]
@gionoa I see that you removed the unit tests that were previously implemented. What was the reasoning behind this?

**Comment by gionoa** [comment]
Hey! I was trying to get it to work on a WatchOS app, and was giving me errors that seemed to be coming from Unit tests so I disabled. But it's definitely a goal to get them back on along with more coverage

**Reply by sjoness** [reply]
Fair enough that makes sense, and I see that you've already got an issue open for creating MockURLSession which will be needed for unit tests 😄 Happy to help on this if needed
